Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,474 @@
+        {
+            if (!InputStringIsValid(personAndBalance))
+            {
+                return default;
+            }
+
+            var array = personAndBalance.Split(',');
+            var newArray = new decimal[array.Length - 1];
+
+            for (int i = 1; i < array.Length; i++)
+            {
+                newArray[i - 1] = ConvertStringToDecimal(array[i]);
+            }
+
+            return newArray;
+        }
+
+        /// <summary>
+        /// Finds the current balance of a single person in array.
+        /// </summary>
+        private static decimal CurrentBalanceForSinglePerson(string personAndBalance)
+        {
+            var balanceList = ConvertStringToDecimalArray(personAndBalance);
+            try
+            {
+                decimal currentBalance = balanceList[^1];
+
+                return currentBalance;
+            }
+            catch (Exception)
+            {
+            }
+            return default;
+        }
+
+        /// <summary>
+        /// Return index of Max value in decimal array.
+        /// </summary>
+        private static int FindDecimalArrayMax(decimal[] inputArray)
         {
-            return "";
+            var max = decimal.Zero;
+            try
+            {
+                max = inputArray[0];
+            }
+            catch (IndexOutOfRangeException)
+            {
+            }
+            var index = 0;
+            for (int i = index; i < inputArray.Length; i++)
+            {
+                if (inputArray[i] > max)
+                {
+                    max = inputArray[i];
+                    index = i;
+                }
+            }
+            return index;
+        }
+
+        /// <summary>
+        /// Return index of Min value in decimal array.
+        /// </summary>
+        private static int FindDecimalArrayMin(decimal[] inputArray)
+        {
+            var min = decimal.Zero;
+
+            try
+            {
+                min = inputArray[0];
+            }
+            catch (IndexOutOfRangeException)
+            {
+            }
+            var index = 0;
+            for (int i = 0; i < inputArray.Length; i++)
+            {
+                if (inputArray[i] < min)
+                {
+                    min = inputArray[i];
+                    index = i;
+                }
+            }
+            return index;
+        }
+
+        /// <summary>
+        /// Returns an formated output of currency (ex. -¤1, ¤4, ¤1002, -¤1001).
+        /// </summary>
+        private static string FormatCurrency(decimal currency)
+        {
+            const char negativeSymbol = '-';
+
+            var formatedCurrency = new StringBuilder();
+
+            if (currency < 0)
+            {
+                currency *= -1;
+                if (DefaultCurrencyLocation == 0)
+                    formatedCurrency.Append(negativeSymbol).Append(DefaultCurrencySymbol).Append(currency);
+                if (DefaultCurrencyLocation == 1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4971928)